files/site.tfvars.in

```diff
@@ -21,6 +21,8 @@ cluster_domain              = "$CLUSTER_DOMAIN"
 cluster_id                  = "ocp$SANITIZED_OCP_VERSION"
 
 dns_forwarders              = "192.168.88.1;$DNS_BACKUP_SERVER"
+chrony_config               = true
```

Comment:
I believe you also need to include the fix from ocp4-upi-kvm submodule.

Reply:
Good point; done.